README.md

```diff
@@ -90,10 +90,16 @@ should be reenabled as soon as possible.]
 
 ## Testing
 ### pytest
+
+Note that you must install the dependencies contained in
+[requirements-test.txt](requirements-test.txt) before running tests. See the explanation in
+[pyproject.toml](pyproject.toml) for details.
+
 [pytest](https://docs.pytest.org/en/stable/) for testing:
 
 ```
 % cd appmap-python
+% pip install -r requirements-test.txt
```

Comment:
Should `.travis.yml` be calling `pip install -r requirements-test.txt`?

Reply:
It doesn't need to, as it's using tox to do the testing.